### DISCUSSION TITLE
set thread to high priority

### DISCUSSION THREAD

**Discussion by yukulele** [FluidSynth/fluidsynth#824] [Q&A] [answered]
When runing fluidsynth I get this message:

```
fluidsynth: warning: Failed to set thread to high priority
```
- What this means?
- how to fix-it?

**Comment by mawe42** [comment, marked as answer]
The warning means that FluidSynth tried to increase a thread priority, but failed. This is usually due to insufficient privileges of the user running FluidSynth. This page has quite a good description on how to configure your system user limits to allow higher priority threads: https://jackaudio.org/faq/linux_rt_config.html

Having said that: depending on your use-case it might not be necessary. For example, if you use FluidSynth to render MIDI to a file, then this warning can safely be ignored.

But if you want to use FluidSynth as a real-time synth, then changing the `rtprio` and `memlock` levels as explained by the above link will definitely help.
